app/src/main/java/space/taran/arknavigator/ui/fragments/ResourcesFragmentHelper.kt

```diff
@@ -93,6 +94,45 @@ fun ResourcesFragment.setupAndShowSelectedResourcesMenu(menuBtn: View) {
                 .show(parentFragmentManager, null)
             popup.popupWindow.dismiss()
         }
+        with(btnIncreaseScore) {
+            isVisible = presenter.allowScoring()
+            setOnClickListener {
+                presenter.onIncreaseScoreClicked()
+            }
+        }
+        with(btnDecreaseScore) {
+            isVisible = presenter.allowScoring()
+            setOnClickListener {
+                presenter.onDecreaseScoreClicked()
+            }
+        }
+        with(btnResetScores) {
+            val selectedSize = presenter.gridPresenter.selectedResources.size
+            isVisible = presenter.allowResettingScores()
+            text = getString(
+                R.string.reset_scores,
+                if (selectedSize == 1) "" else "s"
+            )
+            setOnClickListener {
+                popup.popupWindow.dismiss()
+                if (selectedSize == 0) {
+                    toast(R.string.select_at_least_one_resource)
+                    popup.popupWindow.dismiss()
+                    return@setOnClickListener
+                }
+                ConfirmationDialogFragment.newInstance(
+                    getString(R.string.are_you_sure),
+                    getString(
+                        R.string.resource_scores_erased,
+                        selectedSize,
+                        if (selectedSize == 1) "" else "s"
```

Comment:
:+1:

Reply:
Currently, we only have one language. If support for many languages is added we can always created the resources. I think for now that's simple.

Reply:
Let's not mess with it for this moment.